src/main/frontend/components/block.cljs

```diff
@@ -946,9 +946,7 @@
           (when (and marker
                      (not (string/blank? marker))
                      (not= "nil" marker))
-            {:class (str (string/lower-case marker)
-                         ; " flex flex-row items-center"
```

Comment:
These were not being applied anyway due to the no space, and I think we want block styling anyway for marker blocks?

Reply:
Yes, I think those can be removed.